### PR TITLE
Contributions.md Document Update

### PR DESCRIPTION
**Pull request by kb1lqc**
It was quickly becoming apparent we needed to better define the expectations from a pull request submitting code into Faraday-Software. This document aims to address items like how we should try to commit to Git (atomically) and how we should style Python (PEP8) among other items.

By having a well-defined contributions document we can remove a large number of unnecessary pull request comments largely related to wording, styling, and spelling errors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,121 +1,183 @@
...
-Documentation is mandatory to ensure that all contributors clearly understand how to use, enhance, and fix additions to the project. Faraday uses [Sphinx](http://www.sphinx-doc.org/en/stable/index.html) to provide dynamic documentation and is hosted on [Faraday's Read The Doc's Page](http://faraday-software.readthedocs.io/en/latest/).
+* Spelling errors should be eliminated
+* Try to be as clear and concise as possible
+* Use one space after a period for new sentences
+* When referencing variables, functions, classes, or folders use `code` formatting
 
-* [Sphinx Documentation](http://www.sphinx-doc.org/en/stable/contents.html)
-* [Examples Of Good Python Docstrings](http://www.sphinx-doc.org/en/stable/ext/example_google.html)
+### Documentation<a name="documentation"></a>
+Open source projects strive then they are documented well. This allows everyone to understand what is going on in the code and how to use it. Please follow the following conventions:
```
Comment (kb1lqd) [inline CONTRIBUTING.md]: Typo: ...strive "when" they are documented well....

**Review by kb1lqd — CHANGES_REQUESTED** [review]
Please fixed typo, otherwise good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kb1lqc) [reply CONTRIBUTING.md]: Fixed 😄

**Review by kb1lqd — APPROVED** [review]
Looks good, typo fixed.

**Pull request merged**